src/components/ContentPost/ContentPost.tsx

```diff
@@ -20,32 +14,26 @@ const ContentPost = ({
       href={link}
       className="group flex flex-col overflow-hidden rounded-md w-full bg-grey-100 hover:bg-grey-200 border border-grey-200 hover:border-grey-300 cursor-pointer transition duration-300 shadow-md h-full"
     >
-      <div className="relative group">
-        <Image
-          width={300}
-          height={300}
-          alt=""
-          src={`https:${thumb?.fields?.file?.url || ""}`}
-          className="w-full aspect-7/4 block object-cover object-top transition-transform group-hover:scale-102 duration-300"
-        />
-        {labeled && (
-          <div className="absolute top-4 left-4 rounded-lg px-2 py-0.5 bg-gray-100">
-            <p className="font-semibold text-xs">
-              {
-                POST_TYPES[
-                  type as "additional-content" | "data-panel" | "newsletter"
-                ]
-              }
-            </p>
-          </div>
-        )}
+      <Image
+        width={300}
+        height={300}
+        alt=""
+        src={`https:${thumb?.fields?.file?.url || ""}`}
+        className="w-full aspect-7/4 block object-cover object-top transition-transform group-hover:scale-102 duration-300"
+      />
+      <div className="flex flex-row justify-between bg-gray-200 px-5 py-1">
+        <p className="font-semibold text-xs">
+          {
+            POST_TYPES[
+              type as "additional-content" | "data-panel" | "newsletter"
+            ]
```

Comment:
I think we could add this typing on the interface, what do you think, like this


```js
export interface IPublication {
  title: string;
  link: string;
  thumb: {
    fields: {
      file: {
        url: string;
      };
    };
  };
  type: "additional-content" | "data-panel" | "newsletter";
  date: string;
  description: string;
}
```

With it we don't need this type "casting"

```suggestion
            POST_TYPES[type]
```

Reply:
That's a great idea